### ISSUE TITLE
Selected dot color being overwritten by today dot color

### ISSUE THREAD

**Issue by archansel** [wix/react-native-calendars#1226]
## Description

I think current logic in Dot comp to determined its color is wrong https://github.com/wix/react-native-calendars/blob/master/src/calendar/dot/index.js#L11-L29

In that logic, if I have green background selected color and also green dot color, it will show correct behavior in date except today date, it is because (I think) this https://github.com/wix/react-native-calendars/blob/master/src/calendar/dot/index.js#L26-L28 should have another condition `isToday && !isSelected`

For example:
```
dotColor: green
selectedDotColor: white
selectedDayBackgroundColor: green
```
This will result following (date 12 is today), see its missing white dots
![Screen Shot 2020-07-12 at 13 07 32](https://user-images.githubusercontent.com/816817/87240220-c4afdf00-c441-11ea-87a9-0334eadb6af4.png)

## Expected Behavior

Should show white dots in selected date even for today date

## Observed Behavior

selectedDotColor being overwritten by today dot color which is dotColor if there is no today dot color https://github.com/wix/react-native-calendars/blob/master/src/calendar/dot/style.js#L27-L29

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-calendars`: 1.300.0
* `npm ls react-native`: 0.61.5

Also specify:

1. iOS Simulator & version: iPhone 11 iOS 13.5

**Comment by inkCrazy** [comment]
更新到react-native-calendars@^1.301.0:还是有问题，current day ‘s dotcolor设置无效fail， 总是显示蓝色

**Comment by emilisb** [comment]
Should be fixed by #1239
